Response-Errors.md

```diff
@@ -0,0 +1,123 @@
+| `NOT_FOUND_ERROR` | `3017` | Organization was not found! | Searching organization does not exists | Getting organization |
+| `NOT_FOUND_ERROR` | `3018` | Wallet was not found | You will get this error if you try to activate a wallet which your organization does not hold | Activating vendor wallet |
+| `NOT_FOUND_ERROR` | `3019` | The record you have searched for does not exists | ou will get this error if you pass invalid input, which we can not decode to determine what you are searching for | Every time we can not decode searching filter |
+
+
+### 4. Validation Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `VALIDATION_ERROR` | `4011` | Nonexistent country code | You will get this error if you pass non-existing country code to Vendor, CardHolderInformation | Processing Fiat payment / calculating VAT  |
+| `VALIDATION_ERROR` | `4111` | isCompany should be true/false | You will get this error if you pass non-boolean value to CardHolderInformation.isCompany | Processing payment |
+| `VALIDATION_ERROR` | `4211` | weiAmount must have non-zero value  | You will get this error if your `fundTx.weiAmount` is zero | Creating payment |
+| `VALIDATION_ERROR` | `4311` | Vat number (**your vat number**) is invalid | You will get this error if you pass invalid VAT number in CardHolderInformation | Creating vendor / Processing Fiat payment | 
+| `VALIDATION_ERROR` | `4312` | Vat number is empty | You will get this error if you have vat number as property in CardHolderInformation, but it does not have a value | Creating vendor / Processing Fiat payment |
+
+### 5. Global Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `INTERNAL_SERVER_ERROR` | `5001` | Something went wrong. Contact your support for more information | You will get this error if something internally breaks down | -
+| `BAD_INPUT_ERROR` | `5111` | Wrong input parameters | You will get this error if you pass wrong input parameter | Every time bad input is passed |
+| `REQUESTS_RATE_LIMIT_ERROR` | `5211` | Too many requests, please try again later. | You will get this error if you exceed LimePay API rate limit | Every time rate limit is exceeded |
+
+
+### 6. Payment errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `PAYMENT_ERROR` | `6011` | Payment has already been processed  | You are allowed to create a payment and process it, but not re-process it | Processing payment |
+| `PAYMENT_ERROR` | `6012` | Payment can't be proceed, because shopper is malicious. |  You will get this error if a shopper is flagged as malicious one. This prevents each attempt a shopper do to interact with LimePay API | Processing payment |
```

Comment:
`Payment can't be proceed, because shopper is malicious.`
->
`Payment can't be processed, because shopper is malicious.`
We have typo - `proceed` should be `processed`.


`You will get this error if a shopper is flagged as malicious one. This prevents each attempt a shopper do to interact with LimePay API`
->
`You will get this error if a shopper is flagged as malicious one. You need to contact support once this problem occurs `.